### PR TITLE
Feature/global fixes

### PR DESCRIPTION
**Pull request by nicocadq**
Fixed some issues:
<blockquote>
<ul>
<li>Text Overflows</li>
<li>Move cards inside empty columns</li>
<li>Prevent undefined destination</li>
<li>Fixed AddForm in home view</li>
<li>Back to home if board doesn't exist</li>
</ul>
</blockquote>

### CODE REVIEW

File: src/components/Board/Board.js [hunk 1]
```diff
@@ -40,7 +48,7 @@ const Board = ({ id }) => {
 
   const handleOnDragEnd = (result) => {
     const { destination, source, draggableId } = result;
-    const { droppableId, index } = destination;
+    const { droppableId, index } = destination || {};
```
Comment (eleleiva) [inline src/components/Board/Board.js]: The if below will never be `falsy` if you do this, you could either, remove the `if` or move this to be below the `if` and remove the `|| {}` part, whatever you see more fitting with the logic you are trying to achieve.

File: src/components/Board/Board.js [hunk 2]
```diff
@@ -16,6 +17,13 @@ import AddForm from "../AddForm";
 import styles from "./Board.module.scss";
 
 const Board = ({ id }) => {
+  const history = useHistory();
+  const isBoardExist = useSelector((state) => state.boards[id]);
```
Comment (Rminetti) [inline src/components/Board/Board.js]: I don't think this name is correct, maybe something like boardExists would be better. What do you think?

File: src/components/Board/Board.js [hunk 3]
```diff
@@ -16,6 +17,13 @@ import AddForm from "../AddForm";
...
+
+  if (!isBoardExist) {
+    history.push("/");
+  }
```
Comment (Rminetti) [inline src/components/Board/Board.js]: Maybe you could wrap this on a useEffect son that you can make sure this is evaluated every time the board changes.

File: src/components/Board/Board.js (hunk 2 above)
Comment (eleleiva) [reply src/components/Board/Board.js]: Or `doesBoardExists` maybe.

**Review by eleleiva — APPROVED** [review]

**Review by Rminetti — APPROVED** [review]

**Pull request merged**